docs/contributor/image_mode.md

```diff
@@ -337,6 +337,119 @@ watch sudo oc get pods -A \
     --kubeconfig /var/lib/microshift/resources/kubeadmin/kubeconfig
 ```
 
+## Using Bootc Image Builder (BIB)
+
+The [bootc-image-builder](https://github.com/osbuild/bootc-image-builder), is a containerized tool to
+create disk images from bootc images. You can use the images that you build to deploy disk images in
+different environments, such as the edge, server, and clouds.
+
+### Prepare Build Config File
+A build config is a Toml (or JSON) file with customizations for the resulting image.
+The config file is mapped into the container directory to /config.toml. The customizations are
+specified under a customizations object.
+
+Set variables pointing to secret files that are included in `config.toml` for
+gaining access to private container registries:
+* `PULL_SECRET` file contents are copied to `/etc/crio/openshift-pull-secret`
+  at the post-install stage to authenticate OpenShift registry access
+
+```bash
+PULL_SECRET=~/.pull-secret.json
+IMAGE_NAME=microshift-4.17-bootc
+```
+
+Run the following commands to create the `config.toml` file to be used during
+the bootc image builder process.
+
+```bash
+cat > config.toml <<EOFCONF
```

Comment:
I'm not sure about using the config.tomp file for only embedding kickstart.
What's the main point here to deviate from the kickstart settings in previous sections? 

P.S. I'm not decided on which approach is better, so I'm looking for justification to introduce config.toml notion in this document.

Reply:
@ggiguash Use of kickstart file is required because of providing the required disk layout to work for microshift lvm part otherwise we can directly use BiB on the generated bootc microshift image.

Reply:
I agree that kickstart is required. I'm asking if we need propagate kickstart via config.toml rather than from virt-install command line, like in previous section.

Reply:
Sure, we can use virt-install cli to pass the kickstart but we should mention or link to the doc that how it can also be embedded with iso when using BIB so that created ISO can be used on other hypervisor?

Reply:
Yeap, that would work. Let's add a note pointing to upstread docs [here](https://osbuild.org/docs/bootc/#anaconda-iso-installer-options-installer-mapping)?